src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContex } from '../../peopleContext';
```

Comment:
There is a typo in the context import. It should be `PeopleContext` instead of `PeopleContex`.